docs/src/main/tut/chisel3/cookbook.md

```diff
@@ -384,3 +384,70 @@ However, if we use `@chiselName` then the register previously called `_T` is now
                             |$a
                             |```""".stripMargin))
 ```
+
```

Comment:
```suggestion
### How do I get Chisel to name the results of vector reads properly?
```

Add a header, since this is not really part of the question about `withClockAndReset` / `@chiselName`